### DISCUSSION TITLE
Upgrade of support ticket system pending

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#15487] [Show and tell]
Last year I tried to upgrade the support ticket system (OSTicket) from V1.17.2 to V1.17.3, which I had to revert due to several major issues after upgrade:
- Mail out issues regarding header encoding, which made recipient mail servers deny our mails
- Issues with the spam filtering

Meanwhile those issues should be resolved according to OSTicket release notes.
Furthermore there is now also a version with PHP 8.2 support available, which I would prefer to use to be future proof in case of retirement of older PHP versions by my hosting provider.

I will therefore try to upgrade OSTicket from V1.17.2 to V.1.18.1 in next days as time permits.

For this action I have to take the system offline, perform backup, upgrade and take the system online again.
This might take 1-2 hours.

I will disable mail pulling in beforehand and enable again at the end, so no new tickets or answers should be lost in between.
If however technical issues come up after migration and re-activation I have to restore the backup, thus tickets received after the upgrade will get lost.

I will update this discussion few hours before starting into the process, so that "support helpers" (@eddiemuc and @moving-bits ) will be aware of the update taking place.

**Comment by Lineflyer** [comment]
Will disable support system tonight starting around 8pm CET. 
Please do not login or work on any tickets.

Will notify when done.

**Comment by Lineflyer** [comment]
Done, with medium success:

OSTicket was upgrade from v1.17.2 to only v1.17.5 which is feature equal to v1.18.1 but has PHP 8.0 support.
Version v1.18.1 which needs PHP 8.1+ did not work due to several PHP warnings and errors which I was not able to debug. Looking into OS-Ticket forums some errors are known as server configuration depending, which made me change to v1.17.5.

At least spam filtering issue should therefore be solved.
The mail encoding problem has a workaround: Our sender name must not contain any special chars (so I changed it from "c:geo Support" to "cgeo Support").

Incoming and outgoing mails tested and working (also to GMX).

@eddiemuc and @moving-bits 
Please notify me if you encounter issues; especially if you notice no more incoming tickets/replies over an unusual period of time.

System is online again.

**Reply by eddiemuc** [reply]
At first glance system seems to work fine. Thanks for the upgrade!
